assets/merchant/marketapp.json

```diff
@@ -0,0 +1,46 @@
+{
+    "metadata": {
+        "label": "marketapp",
+        "category": "merchant",
+        "subcategory": "",
+        "website": "https://marketapp.ws",
+        "description": "The most profitable marketplace on TON aggregates all including non-standard deals",
```

Comment:
Great job! 
IMO I'd change this description to something more neutral because "the most profitable" maybe not true in several months but this label may not be changed never again...